plugins/salesforce/komand_salesforce/util/api.py

```diff
@@ -368,11 +367,16 @@ def get_error(response: str) -> Tuple[str, str, Any]:
                     401,
                 )
             if error_desc:
+                status_code = ""
```

Comment:
I am just wondering why it's defined here as `""`, but at line `372` it's setup as `500` integer. I mean more the type mismatch here 😄

Reply:
this is how it was originally, it leaves it as a blank value so that the status code returned in the response can override it if its empty